nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -71,78 +71,81 @@ We need to set the `NAUTOBOT_ROOT` environment variable for the `nautobot` user
 
 Run this command to update `~/.bashrc` for `nautobot` so that anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
 
-```no-highlight
+```no-highlight title="Add NAUTOBOT_ROOT to nautobot user bashrc file"
 echo "export NAUTOBOT_ROOT=/opt/nautobot" | sudo tee -a ~nautobot/.bashrc
 ```
 
 ## Sudo to nautobot
 
 It is critical to install Nautobot as the `nautobot` user so that we don't have to worry about fixing permissions later.
 
-```no-highlight
+```no-highlight title="Log into the nautobot user"
 sudo -iu nautobot
 ```
 
-Observe also that you can now echo the value of the `NAUTOBOT_ROOT` environment variable that is automatically set because we added to `.bashrc`:
+??? note "Validate the NAUTOBOT_ROOT variable"
+    Observe also that you can now echo the value of the `NAUTOBOT_ROOT` environment variable that is automatically set because we added to `.bashrc`:
 
-```no-highlight
-echo $NAUTOBOT_ROOT
-```
+    ```no-highlight title="Verify Nautobot Root"
+    echo $NAUTOBOT_ROOT
+    ```
 
-Example output:
+    ??? example "Example NAUTOBOT_ROOT output"
```

Comment:
Maybe a little redundant to have a collapsed section inside a collapsed section?